languages/elixir/exercises/concept/maps/.docs/instructions.md

```diff
@@ -0,0 +1,112 @@
+In this exercise you're going to write a module to create and manipulate a map or high scores.
+
+You have 7 functions to implement, all related to returning and manipulating a map of high score data.
+
+### 1. Define a new high score map
+
+Define the `HighScore.new/0` function which doesn't take any parameters and returns new empty map of highscores.
+
+```elixir
+HighScore.new()
+# => %{}
+```
+
+### 2. Add players to the high score map
+
+Define `HighScore.add_player/3`, which is a function which takes 3 parameters:
+
+- The first parameter is the map of scores.
+- The second parameter is the name of a player as a string.
+- The third parameter is the score as an integer. The parameter is optional, implement the third parameter with a default value.
```

Comment:
Maybe explicitly specify what the default value is?

Reply:
```suggestion
- The third parameter is the score as an integer. The parameter is optional, implement the third parameter with a default value of 0.
```